examples/c/5_series/ahrs/README.md

```diff
@@ -0,0 +1,187 @@
+| `PORT_NAME`        | `"COM1"` (Windows)<br>`"/dev/ttyACM0"` (Unix) | Serial port for device communication |
+| `BAUDRATE`         | `115200`                                      | Communication baud rate              |
+| `SAMPLE_RATE_HZ`   | `1`                                           | Data output rate in Hz               |
+| `RUN_TIME_SECONDS` | `30`                                          | Example runtime duration             |
+
+## Key Functions
+
+### Device Setup
+- `initialize_device()` - Establishes communication, validates device connection, and loads defaults
+- `capture_gyro_bias()` - Captures and applies gyroscope bias compensation
+- `initialize_filter()` - Initializes the attitude filter with magnetometer as the heading source
+
+### Message Configuration
+- `configure_filter_message_format()` - Configures filter/attitude data output including:
+    - Filter timestamp
+    - Filter status
+    - Euler angles (roll, pitch, yaw)
+
+### Data Display
+- `display_filter_state()` - Displays navigation filter operating mode changes
+
+### Communication Interface
+- Uses the `mip_interface` struct for device communication
+- Serial connection handled by `serial_port`
+
+## Data Handling
+
+This example demonstrates traditional C programming patterns:
+- **Manual Parsing**: Direct parsing of incoming MIP packets using the MIP parser
+- **Callback Functions**: Function pointer-based callbacks for data processing
+- **Explicit Memory Management**: Manual buffer and resource management
+- **Type Safety**: Uses C structs and enums for data type safety
+- **Error Codes**: Command results handled through return codes and error checking
+
+## C Implementation Features
+
+This example showcases:
+- **MIP Interface**: Core C interface for device communication (`mip_interface`)
+- **Serial Port Management**: Low-level serial port operations
+- **Parser Integration**: Direct use of the MIP packet parser
+- **Memory Safety**: Careful buffer management and bound checking
+- **Portability**: Cross-platform compatibility (Windows/Unix)
+
+## Custom Communication Functions
+
+The example implements custom communication handlers:
+- **Send Function**: `mip_interface_user_send_to_device()` handles outgoing data
+- **Receive Function**: `mip_interface_user_recv_from_device()` manages incoming data
+- **Timeout Handling**: Configurable timeouts for reliable communication
+
+## Filter Data Output
+
+The example streams the following filter data:
+
+### TOW Data
+- **Units**: seconds
+- **Description**: Time of Week - GPS time reference
+- **Format**: Floating-point timestamp value
+
+### Euler Angles Data
+- **Units**: radians
+- **Description**: Orientation expressed as Euler angles
+- **Format**: [Roll, Pitch, Yaw] angle vector
+
+## Streaming Output Format
+
+The example displays filter data in the following format:
+```
+TOW = 123456.789     Euler Angles = [ 0.012345, -0.067890,  1.234567]
+```
+
+## Usage
```

Comment:
For this and all other example READMEs, I would put this either right before or right after Configuration. The rest of this README is nice for context, but most users want to see "Usage" first